fp-proj/src/app/app.component.html

```diff
@@ -1,2 +1,8 @@
 <app-navigation></app-navigation>
-<router-outlet></router-outlet>
+
+  <div *ngIf="loading; else elseBlock" class="spinner-border" role="status">
+    <span class="sr-only">Loading...</span>
+  </div>
+  <ng-template #elseBlock><router-outlet></router-outlet></ng-template>
```

Comment:
Auuuuuuuuuuuć :)
to bardzo nieoptymalne rozwiazanie :) router-outlet tez uzywa ng-template w srodku i robi sie takie zagniezdzenie templatow - obawiam sie ze kiedys moze sie to rozpieprzyc i narobic problemow :) no i na pewno nie jest to najszybsze rozwiazanie chociazby dla changeDetectora ktory bedzie ci usuwal widoki przy kazdym wlaczeniu loadera.

duzo lepszym rozwiazaniem jest umieszczenie loadera na warstwie i zaslanianie zawartosci